webpack/assets/javascripts/react_app/components/common/TypeAheadSelect/TypeAheadSelect.fixtures.js

```diff
@@ -0,0 +1,39 @@
+import Immutable from 'seamless-immutable';
+
+export const id = 'test_type_ahead_select';
+export const options = ['option1', 'option2'];
+export const selections = ['option2'];
+export const multiple = true;
+export const allowNew = true;
+export const clearButton = true;
+export const placeholder = 'Inspirational placeholder';
+export const initialState = Immutable({ typeAheadSelect: {} });
```

Comment:
If they're not imported anywhere, these can all just be `const` instead of `export const`